addons/mail/static/src/models/composer/composer.js

```diff
@@ -102,6 +102,20 @@ function factory(dependencies) {
             }
         }
 
+        /**
+         * Focus a specific composer and remove focus from all others.
+         * @param {mail.composer} composer
+         */
+        static focus(composer) {
+            const allComposers = this.env.models['mail.composer'].all();
+            for (const otherComposer of allComposers) {
+                if (otherComposer !== composer && otherComposer.hasFocus) {
+                    otherComposer.update({ hasFocus: false });
+                }
+            }
+            composer.update({ hasFocus: true });
+        }
+
```

Comment:
Why not instance method?

Reply:
I thought it was better to have a static one as it modifies multiple instance but obviously it can also be an instance one if you prefer

Reply:
I would avoid `static` methods as much as possible, because they do not work well with many things (no `this.async()`, no field integration, etc...).

Reply:
Alright